virttest/utils_package.py

```diff
@@ -106,14 +106,21 @@ def operate(self, timeout, default_status, internal_timeout=2):
                 need = True
             if need:
                 cmd = self.cmd + pkg
-                if self.session.cmd_status(cmd, timeout, internal_timeout):
+                status, output = self.session.cmd_status_output(cmd,
+                                                                timeout,
+                                                                internal_timeout)
+                if status:
+                    logging.error("Operate %s with %s "
+                                  "failed:%s", pkg, cmd, output)
```

Comment:
pkg is already part of cmd from line 108, do we really want to output it twice?

Reply:
@kamilvarga , ok, I just copied from original codes. Now I have refined it.